config/initializers/health_monitor.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+Rails.application.config.after_initialize do
+  HealthMonitor.configure do |config|
+    config.cache
```

Comment:
What is the cache monitoring here? disk access? I don't think dpul has any special cache setup?

Reply:
Okay it just checks that it can add and then fetch something from the cache. https://github.com/lbeder/health-monitor-rails/blob/master/lib/health_monitor/providers/cache.rb

I don't really think we need it for dpul but I guess it doesn't hurt!